workspaces/copilot/plugins/copilot/src/components/Table/LanguagesBreakdownTable.tsx

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
```

Comment:
Just noticing that in some cases we use from `@mui` and others from `@material-ui`, from a forward looking perspective it might make more sense to just use everything from `@mui`, you'll have less work later on porting this all over.

Reply:
I must admit that this adjustment caused quite a bit of headache. As you’ll see in the commit, I needed to make some tweaks to ensure everything worked as expected.